### DISCUSSION TITLE
Validity of medications administered in the PRESCRIPTIONS table

### DISCUSSION THREAD

**Discussion by rielmvp** [MIT-LCP/mimic-code#1603] [MIMIC-III]
Good day,

In the MIMIC documents, it was mentioned that the PRESCRIPTIONS table is composed of " Medications ordered, and not necessarily administered, for a given patient". Therefore, I'm wondering if there is any way to check or validate whether the specific prescription was given during the associated ICU stay.

I understand that drugs administered are also available in the D_ITEMS table, but I noticed that some drugs, such as Ibuprofen, only have records in the PRESCRIPTION table but not in the D_ITEMS table, while other drugs such as Amphotericin B are found in both PRESCRIPTIONS and D_ITEMS table. It is because of this that I want to ask about any possible methods to check the validity of whether or not the ordered prescriptions were actually administered. Any information would be much appreciated.

Best regards,

**Comment by alistairewj** [comment]
Your best bet is to check the eMAR table. Here's a rough guide:

- d_items / inputevents - Administrations, charted by nursing staff, only
available in the ICU, only infusions, and a preselected subset of drugs
(and so there are some notable omissions).
- prescriptions - Orders, available hospital wide, non-formulary orders are
unstructured but present
- eMAR / emar_detail - Administrations, from barcode scanning medications,
hospital wide but only for more recent patients

For the very common infusions, done in the ICU, inputevents has excellent
data capture. For oral medications, eMAR would be the only way to capture
administrations. To be honest, I don't know how PRN medications are
captured in eMAR. Many studies approximate administration with
prescriptions for these reasons.

Of course if you are using MIMIC-III, then eMAR doesn't exist (it was
implemented after the data collection), so your choice is simpler!

**Reply by rielmvp** [reply]
Thanks for the reply. It certainly shed some light on how to differentiate administrations and prescriptions.

If I may ask another question @alistairewj, you mentioned that " Many studies approximate administration with
prescriptions for these reasons.". May I know some studies that come to mind or some studies that you know which definitely approximate administration with prescriptions for these reasons?

Best regards,
